x/incentive/spec/03_keeper.md

```diff
@@ -0,0 +1,49 @@
+<!--
+order: 3
+-->
+
+# Keepers
+
+## Reward pool address for LPs
+
+There's constant reward pool address per liquidity pool id which will be distributed per epoch.
+
+```go
+    func GetLPRewardsPoolAddress(poolId uint64) {
+        return authtypes.NewModuleAddress(fmt.Sprintf("lp_rewards_pool_%d", poolId))
+    }
+```
+
+DEX revenue and margin revenue to be distributed to specific pool liquidity providers is sent to the address and incentive module distribute the those tokens to relevant pool LPs.
+
+## Elys token to USDC conversion
```

Comment:
Gas fees will be Elys and we will need to convert it to USDC before distributing. I think we have agreed on the fact that all the DEX revenues will be collected to the reward wallet in the form of USDC. Meaning inside incentive module, we don't need to do any conversions on the rewards collected as it is already USDC. So I think we will just need to distribute them appropriately.